test/unit/FixedLatencyMemoryInterfaceTest.cc

```diff
@@ -26,8 +25,24 @@ class FixedLatencyMemoryInterfaceTest : public testing::Test {
   simeng::FixedLatencyMemoryInterface memory;
 };
 
-// Test that we can read data and it completes after a number of cycles.
-TEST_F(FixedLatencyMemoryInterfaceTest, FixedReadData) {
+// Test class definition for a fixed latency memory interface with a latency of
```

Comment:
Not a fan of this solution. Can we please change it to a similar solution to that used in `SimEng/test/unit/pipeline/LoadStoreQueueTest.cc` (the `getQueue` function)

Reply:
Updated